tmol/score/lk_ball/potentials/lk_ball.hh

```diff
@@ -1178,7 +1178,7 @@ TMOL_DEVICE_FUNC void lk_ball_atom_derivs_full(
   Eigen::Matrix<Real, n_lk_ball_score_types, 1> dTdV_local;
 
   for (int i = 0; i < n_lk_ball_score_types; ++i) {
-    dTdV_local[i] = dTdV[block_pair_dat.pose_ind][i];
+    dTdV_local[i] = dTdV[i][block_pair_dat.pose_ind];
```

Comment:
I'm just a little surprised this is showing up as a difference for the merge. This change has already been made in trunk.